rust/tvm-graph-rt/tests/test_tvm_basic/build.rs

```diff
@@ -48,10 +48,6 @@ fn main() -> Result<()> {
         obj_file.exists(),
         "Could not build tvm lib: {}",
         String::from_utf8(output.stderr)?
-            .trim()
-            .split("\n")
-            .last()
-            .unwrap_or("")
```

Comment:
Wonder how this came up. Just for readability?

Reply:
Oh, that's really weird.  I'm guessing it was from bouncing over to the PR branch of https://github.com/apache/tvm/pull/16183, which touched a number of the FFI bindings.  I've removed this delta from the PR.